package exceptionHandling;

import java.io.InputStream;
import java.util.InputMismatchException;
import java.util.Scanner;

public class testException {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try{
		Scanner s=new Scanner(System.in);
		int a=s.nextInt();
		int b=s.nextInt();
		System.out.println(a+b);
		}
		catch(InputMismatchException e)
		{
			System.out.println("Enter the number");
		}
	}

}
